src/utils.ts

```diff
@@ -239,13 +239,16 @@ export const compareTextWithArray = (
     }
     if (ignoreCase) {
         actual = actual.toLowerCase()
-        expectedArray = expectedArray.map((item) => (item instanceof RegExp ? item : item.toLowerCase()))
+        expectedArray = expectedArray.map((item) => (item instanceof RegExp || typeof item !== 'string' ? item : item.toLowerCase()))
```

Comment:
This will be sufficient
```suggestion
        expectedArray = expectedArray.map((item) => (typeof item !== 'string' ? item : item.toLowerCase()))
```

because:

```
> const regex = new RegExp('')
> typeof regex !== 'string'
< true
```

Reply:
sure, careless mistake. Thanks